BlocQuery/BQAnswerTableViewController.m

```diff
@@ -259,6 +262,21 @@ - (void)didPressAddAnswerButton:(id)sender
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
     [super tableView:tableView didSelectRowAtIndexPath:indexPath];
     
+    //locate answer tapped
+    BQAnswer *temp = [self.objects objectAtIndex:indexPath.row];
+    
+    //build query for the user who wrote that answer
+    PFQuery *tempQuery = [BQUser query];
+    [tempQuery whereKey:@"username" equalTo:temp.userName];
+    
+    //load query into temporary user
+    BQUser *tempUser = [tempQuery findObjects][0];
+    
+    //create new profile screen for that user
+    BQProfileViewController *newProfileVC = [[BQProfileViewController alloc]initWithUser:tempUser];
+    
+    //go to that screen
+    [self.navigationController pushViewController:newProfileVC animated:YES];
```

Comment:
I know that this is temp and old data which is potentially garbage, but our current set of data in Parse gives us some good practice in defensive coding... at the moment, on one of our (null) users, we get the following crash:
```
2015-02-10 22:32:45.359 BlocQuery[9858:1275528] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Cannot do a comparison query for type: (null)'
*** First throw call stack:
(
	0   CoreFoundation                      0x00000001084cef35 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x0000000108138bb7 objc_exception_throw + 45
	2   CoreFoundation                      0x00000001084cee6d +[NSException raise:format:] + 205
	3   BlocQuery                           0x00000001053c0a48 +[PFInternalUtils assertValidClassForQuery:] + 333
	4   BlocQuery                           0x00000001053912fe -[PFQuery whereKey:equalTo:] + 86
	5   BlocQuery                           0x000000010527be52 -[BQAnswerTableViewController tableView:didSelectRowAtIndexPath:] + 322
```
We'll want to verify that `temp` and `temp.userName != nil` in this scenario:
```
    [tempQuery whereKey:@"username" equalTo:temp.userName];
```